components/PaymentButton.js

```diff
@@ -0,0 +1,18 @@
+import { usePayment, useTrip } from '../hooks';
```

Comment:
magic

Reply:
Yup, I used to feel like that before I actually started using hooks; Clarke's third law remains in full effect...